views/index.html.twig

```diff
@@ -67,9 +66,9 @@
                         <div class="card">
                             <img src="{{ src_images_products ~ producto.imagen }}" class="card-img-top" alt="...">
                             <div class="card-body">
-                                <h5 class="card-title"><a href="/front.php/product/{{ producto.id }}">{{ producto.name }}</a></h5>
+                                <h5 class="card-title"><a href="{{ url_generator.generate('product', { 'id': producto.id }) }}">{{ producto.name }}</a></h5>
                                 <h5 id="price"> $ {{ producto.price }}</h5>
-                                <a class="btn btn-primary btn-sm" href="/front.php/add_cart?id={{ producto.id }}">Add to cart</a>
+                                <a class="btn btn-primary btn-sm" href="{{ url_generator.generate('add_cart', { 'id': producto.id }) }}">Add to cart</a>
```

Comment:
@Rodolphus85 Acá además de usar el UrlGenerator, le agregué el id a la ruta add to cart para después poder agarrarlo desde el controller sin usar $_GET.